src/views/hostnetworkconfiguration/components/TopologySidebar/InterfaceDrawer/NeighborInformations.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { FC } from 'react';
+import {
+  CHASSIS_ID,
+  NodeNetworkConfigurationInterfaceLLDPNeighbor,
+} from 'src/nmstate-types/custom-models/NodeNetworkConfigurationInterfaceLLDP';
+
+import { Stack, StackItem, Tooltip } from '@patternfly/react-core';
+import { useNMStateTranslation } from '@utils/hooks/useNMStateTranslation';
+import {
+  getChassisId,
+  getDescription,
+  getIee8021Vlans,
+  getPortId,
+  getSystemDescription,
+  getSystemName,
+} from '@utils/neighbors/getters';
+
+const NeighborInformations: FC<{ neighbor: NodeNetworkConfigurationInterfaceLLDPNeighbor }> = ({
+  neighbor,
+}) => {
+  const { t } = useNMStateTranslation();
+
+  const chassisIdDescription = getDescription(neighbor, CHASSIS_ID);
+  const portId = getPortId(neighbor);
+  const chassisId = getChassisId(neighbor);
+  const systemName = getSystemName(neighbor);
+  const systemDescription = getSystemDescription(neighbor);
+  const iee8021Vlans = getIee8021Vlans(neighbor).sort((a, b) => a.vid - b.vid);
+
+  return (
+    <Stack>
+      {chassisId && (
+        <StackItem>
+          <strong>{chassisIdDescription}</strong>: {chassisId}
+        </StackItem>
+      )}
+
+      {portId && (
+        <StackItem>
+          <strong>{t('Port')}</strong>: {portId}
+        </StackItem>
+      )}
+      {systemName && (
+        <StackItem>
+          <strong>{t('System Name')}</strong>: {systemName}
```

Comment:
These titles here should use sentence case, so the words following the first should start with a lower-case character.

Reply:
These still need to be changed.